packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/AppConfiguration.kt

```diff
@@ -94,12 +96,28 @@ public interface AppConfiguration {
         // TODO We should use a multi threaded dispatcher
         //  https://github.com/realm/realm-kotlin/issues/501
         private var dispatcher: CoroutineDispatcher = singleThreadDispatcher("dispatcher-$appId")
-
+        private var encryptionKey: ByteArray? = null
         private var logLevel: LogLevel = LogLevel.WARN
         private var removeSystemLogger: Boolean = false
         private var syncRootDirectory: String = appFilesDirectory()
         private var userLoggers: List<RealmLogger> = listOf()
 
+        /**
+         * Sets the encryption key used to encrypt the user metadata Realm only. Individual
+         * Realms need to use [SyncConfiguration.Builder.encryptionKey] to encrypt them.
+         *
+         * @param key a 64 byte encryption key.
+         * @return the Builder instance used.
```

Comment:
I added `@return` to the docs; however, the other methods using `apply` do not have this explicitly in the docs. What are your thoughts on this?

Reply:
Yeah, I noticed that we missed adding these before. We should probably add it to all the other ones as well. But this is fine 👍